utils/node_local_test.py

```diff
@@ -417,7 +418,8 @@ def stop(self):
         # When parsing the server logs do not report on memory leaks
         # yet, as if it fails then lots of memory won't be freed and
         # it's not helpful at this stage to report that.
-        # TODO: Remove this block when daos_server shutdown works.
+        # TODO:                              # pylint: disable=fixme
```

Comment:
We do want to fix this, why disable it?

Reply:
I separated the TODO: statement with 2 statements or will causing a pylint error.
#TODO: 
# Remove this block when daos_server shutdown works.

(I have restored it back to the original 1 line.)